# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/aop/hotfix-cross.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Ambassador-Of-Pain.git/Makefile.vanilla	2025-01-29 18:11:17.213333158 +0100
+++ Ambassador-Of-Pain.git/Makefile	2025-01-29 18:12:10.893333162 +0100
@@ -1,17 +1,17 @@
 
-LDFLAGS = -lncurses
+prefix=/usr
 
 aop: aop.o
-	$(CC) -o aop aop.c $(LDFLAGS)
+	$(CC) aop.c -lncurses -o aop
 
 aop.o: aop.c
 	$(CC) -c aop.c
 
 install: aop
-	install -d /usr/local/bin
-	install -d /usr/local/share/aop
-	install -m 755 aop /usr/local/bin/aop
-	install -m 644 aop-level-*.txt /usr/local/share/aop/
+	install -d $(DESTDIR)$(prefix)/bin
+	install -d $(DESTDIR)$(prefix)/share/aop
+	install -m 755 aop $(DESTDIR)$(prefix)/bin/aop
+	install -m 644 aop-level-*.txt $(DESTDIR)$(prefix)/share/aop/
 
 clean:
 	rm -f aop core* *~
